usbmon.c

```diff
@@ -160,20 +175,20 @@ int main(int argc, char **argv) {
     while(1) {
         FD_ZERO(&fds);
         FD_SET(fd, &fds);
-        ti.tv_sec = 10; // TODO(tenox): make this in to a flag
+        ti.tv_sec = tv_sec; // TODO(tenox): make this in to a flag (done)
```

Comment:
remove todo comment instead of adding done

Reply:
removed it